includes/definitions/discovery/fortigate.yaml

```diff
@@ -6,68 +6,77 @@ modules:
         hardware: ENTITY-MIB::entPhysicalModelName.1
     processors:
         data:
-            -
-                oid: fgSysCpuUsage
-                num_oid: '.1.3.6.1.4.1.12356.101.4.1.3.{{ $index }}'
-                type: fortigate-fixed
+            - oid: fgSysCpuUsage
+              num_oid: ".1.3.6.1.4.1.12356.101.4.1.3.{{ $index }}"
+              type: fortigate-fixed
```

Comment:
Hi @epacke 
Please avoid changing the structure of the YAML files. We have most of them following this structure.
Thanx

Reply:
Sorry, that was intentional